app/views/self_service/offices/_offices_table.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="table-wrapper">
+  <table class="offices-table">
```

Comment:
Should this also use `.datatable-default` to make it a scrollable table or is it unlikely to contain that many values?